include/nuttx/sched_note.h

```diff
@@ -386,6 +390,13 @@ struct note_irqhandler_s
   uint8_t nih_irq;              /* IRQ number */
 };
 
+struct note_wdog_s
+{
+  struct note_common_s nmm_cmn;      /* Common note parameters */
```

Comment:
What does meaning of **cmm** ?

Reply:
`nmm` means `note common memory` according to this commit.

https://github.com/apache/nuttx/pull/13847/commits/530783008fe66f40c2e2c1bc328df014eca8864a

Reply:
May rename to `nwd`?  And correspondingly `nhp` for heap?